plugins/katello/3.14/installation/index.md

```diff
@@ -170,6 +170,65 @@ arguments. Multiple entries can be entered via
         reverse: true
         contact: hostmaster.example.com.
 
+## Tuning options
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from Foreman environments deployed at scale in large user environments.
+
+When the foreman-installer is run, it is deployed with a predefined tuning profile named `default`. The definition of the `default` profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
```

Comment:
This may be a bit misleading. As it's written now it implies when `medium` is selected, `common` is no longer applied which isn't true. Common is used for all tuning. In a mathematical way: `tuning = common + profile`. Technically Hiera looks at `foreman.hiera/tuning/sizes/default.yaml` and then `foreman.hiera/tuning/common.yaml`. When a file doesn't exist, Hiera just continues.

Reply:
This is true for now unless we make an empty default yaml file. So for me it reads OK.

Reply:
I disagree. `common` is also used with `medium`. The current text implies you **only** need to look at it when you use default. Given later on we recommend users to look at their tuning, they may keep redundant tuning in `custom-hiera.yaml`.

I'd split it up and leave out how to find the exact tuning till later.

```suggestion
When the foreman-installer is run, it is deployed with a predefined tuning profile named `default`. Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
```

Reply:
I kinda disagree with that idea as this goes back to my whole problem with the naming in the first place. The default tuning profile is common. Common is the default. Common is *always* applied so it is the defaults. When looking at say medium, I should be pointed at the medium profile as thats what makes medium _different_ than the default.

Reply:
Then I have a design question on the real necessity of `foreman.hiera/tuning/common.yaml` file.
I think we should do one of the two options:
Option 1
**********
1. Strip out the changing parameters of  `foreman.hiera/tuning/common.yaml` and move them to a new `foreman.hiera/tuning/sizes/default.yaml` file.
2. And mention in docs that tuning will be always applied in this order:
   - common.yaml
   - `selected tuning profile`. For e.g., medium.yaml  (Note any configuration here overrides default .yaml)
   - custom-hiera.yaml always precedes all other tuning profiles above.

Option 2
**********
- Rename `foreman.hiera/tuning/common.yaml` to `default.yaml` and move it to `foreman.hiera/tuning/sizes`. Note: No common.yaml anymore.
- And mention in docs that tuning will be always applied in this order:
   - default.yaml
   - `selected tuning profile`. For e.g., medium.yaml  (Note any configuration here overrides default .yaml)
   - custom-hiera.yaml always precedes all other tuning profiles above.